### ISSUE TITLE
svg preview broken

### ISSUE THREAD

**Issue by Janpot** [a85/POSTMan-Chrome-Extension#2001]
Trying to fetch an svg, the image in the preview window is broken:

<img width="971" alt="screen shot 2016-04-17 at 13 01 09" src="https://cloud.githubusercontent.com/assets/2109932/14586568/9e5eac8c-049c-11e6-91c7-b3cf7f8b54d1.png">

**Comment by abhijitkane** [comment]
--Also from issue 2077--

```
Please include the following:

Postman Version: 4.2.1
App (Chrome app or Mac app): Chrome 50.0.2661.102
OS details: win / x86-32
Is the Interceptor on and enabled in the app: no
I have a get-request that returns an SVG.

Response headers displayed in postman:

Content-Length →7733
Content-Type →image/svg+xml; charset=utf-8

The body is not displayed. There is only a broken icon. Also Copy to clipboard and Save response don't work.
When I do the call in Chrome, the Image is displayed well.
My expectation is that the body was displayed as text. Additionally the rendering of the image in an extra tab would be great.
```

**Comment by abhijitkane** [comment]
@Janpot This has been fixed in v4.4.2 of the app (Chrome only). Can you update (https://www.getpostman.com/docs/updating) and check?

**Comment by Janpot** [comment]
Works 👍
